tensorflow-ops/src/TensorFlow/Gradient.hs

```diff
@@ -667,6 +668,8 @@ opGrad "LabelClasses" _ _ _ = [Nothing, Nothing]
 opGrad "LabelWeights" _ _ _ = [Nothing]
 opGrad "Size" _ _ _ = [Nothing]
 opGrad "ZerosLike" _ _ _ = [Nothing]
+opGrad "Fill" _ _ [dz] = [Nothing, Just $ sum dz rx]
+  where rx = vector [0] :: Tensor Build Int32
```

Comment:
Thanks, I see there is a function `rangeOfRank` in `Gradient.hs`, I'll use that.